src/components/UI/navbar/index.js

```diff
@@ -47,21 +47,23 @@ const Navbar = () => {
             setIsLoggedIn(false);
             throw new Error(`${response.status} (${response.statusText})`);
           }
+
           return response.json();
         })
         .then((responseJson) => {
-          if (!responseJson.incompleteUserDetails) {
-            setIsLoggedIn(true);
-            setUserData({
-              userName: responseJson.username,
-              firstName: responseJson.first_name,
-              profilePicture: responseJson.picture.url?.DEFAULT_AVATAR,
-            });
+          if (responseJson.incompleteUserDetails) {
+            window.open('https://my.realdevsquad.com/signup', '_blank');
```

Comment:
Could you look into the importance of `no-referrer` and `no-opener` here please
https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener

Reply:
Yes , noopener makes `window.opener= null`(of new tab) which nullify the reference of parent tab from the new tab opened which helps in securing the parent site.

Reply:
Do we need to pass in some information here?